Foxtrot/Tests/ExtractorTest.cs

```diff
@@ -12,47 +12,262 @@
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1015.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: @"/optimize",
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1016.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CheckerErrors.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+            }
+        }
+
+        public static IEnumerable<object> ExtractorTest
+        {
+            get
             {
-                try
-                {
-                    string sourceFile = (string)TestContext.DataRow["Name"];
-                    string options = (string)TestContext.DataRow["Options"];
-                    TestContext.WriteLine("FAILED: Name={0} Options={1}", sourceFile, options);
-                }
-                catch { }
+                return Enumerable.Range(0, ExtractorTestData.Count()).Select(i => new object[] { i });
```

Comment:
Hm.. (and this one third "Hm.." for this review). As I mentioned, I used NUnit and all of this is not an issue there.

But closed for now:(